perfeed/git_providers/github.py

```diff
@@ -210,7 +210,99 @@ async def list_pr_numbers(
 
         return all_prs
 
+    async def _fetch_pr_numbers(self, repo_name: str, start_date: datetime, end_date: datetime):
```

Comment:
Please hold on to use this function as I have a patch to fix an issue (that would cause this function a long time to finish). I'll open a PR soon for you to rebase on top of. Thanks!

Reply:
There you go. PTAL https://github.com/Perfeed/perfeed/pull/17